docs/agents/chopper_mc2000b.rst

```diff
@@ -0,0 +1,56 @@
+.. highlight:: rst
+
+.. _chopper_mc2000b_agent:
+
+========================
+Chopper MC2000B Agent
+========================
+
+The MC2000B Chopper Agent is an OCS agent which helps monitor input and output
+frequencies of the chopper, and sends commands to set the frequency of the chopper,
+as well as other features such as the bladetype and reference modes of the device.
+
+Dependencies
```

Comment:
Mention here that this needs to run on Windows. Any insight into setting that up is also useful, especially since this doesn't have a Docker image. This is our only Windows Agent, so details like setting the `$OCS_CONFIG_DIR` appropriately would be useful.